ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -19,6 +124,307 @@ const triggerSet: TriggerSet<Data> = {
+      netRegexJa: NetRegexes.startsUsing({ id: ['67E5', '67E6'], source: '秘紋', capture: true }),
+      condition: (data) => data.exoterikosCounter === 0,
+      delaySeconds: 0.1,
+      promise: refreshCombatantsPromise,
+      alertText: (data, matches, output) => {
+        data.storedSigils.push({
+          x: parseFloat(matches.x),
+          y: parseFloat(matches.y),
+          id: matches.sourceId,
+        });
+
+        if (data.storedSigils.length < 2)
+          return;
+
+        ++data.exoterikosCounter;
+
+        let dir1;
+        let dir2;
+        for (const sigil of data.storedSigils) {
+          if (sigil.x < 100)
+            dir1 = 'west';
+
+          if (sigil.x > 100)
+            dir1 = 'east';
+
+          if (sigil.y < 100)
+            dir2 = data.sigilMap[sigil.id] === 'Blue' ? 'north' : 'south';
+
+          if (sigil.y > 100)
+            dir2 = data.sigilMap[sigil.id] === 'Red' ? 'north' : 'south';
+        }
+
+        if (!dir1 || !dir2) {
+          console.error(`Exoterikos 1: Unknown layout, ${JSON.stringify(data.storedSigils)}`);
+          data.storedSigils = [];
+          return;
+        }
+
+        data.storedSigils = [];
+        return output[dir2 + dir1]!();
+      },
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Algedon NE/SW',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67EC', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67EC', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67EC', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67EC', source: 'ゾディアーク' }),
+      alertText: (_data, _matches, output) =>
+        output.combo!({
+          dir1: output.northeast!(),
+          dir2: output.southwest!(),
+        }),
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Algedon NW/SE',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67ED', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67ED', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67ED', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67ED', source: 'ゾディアーク' }),
+      alertText: (_data, _matches, output) =>
+        output.combo!({
+          dir1: output.northwest!(),
+          dir2: output.southeast!(),
+        }),
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Paradeigma 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67BF', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67BF', source: 'ゾディアーク' }),
+      condition: (data) => data.paradeigmaCounter === 0,
```

Comment:
This is maybe a silly request, but do you think it's worth having the `paradeigmaCounter` match the `id` here? It seems harder to get wrong if they're the same.